core/snippets/snippets_insert_raw_text.py

```diff
@@ -16,9 +16,59 @@ class Stop:
     col: int
 
 
+stop_stack: list[Stop] | None = None
+
+
+def update_stop_information(stops: list[Stop]):
+    global stop_stack
+    if len(stops) > 1:
+        stop_stack = stops
+        stop_stack.reverse()
+    else:
+        stop_stack = None
```

Comment:
Is there reason you can't just assign an empty list?

Reply:
I decided to refactor the code to always have the stop stack be a list to simplify the logic.